### PR TITLE
Add The Cycle: Frontier Wiki

### PR DESCRIPTION
**Pull request by VeryGreatFrog**

### CODE REVIEW

File: data/sitesEN.json [hunk 1]
```diff
@@ -834,5 +834,16 @@
     "destination_content_path": "/wiki/",
     "destination_platform": "mediawiki",
     "destination_icon": "zeldapedia.png"
+  },
+  {
+    "id": "en-zelda",
```
Comment (KevinPayravi) [inline data/sitesEN.json:839]: Please update id (can be `en-cyclefrontier`)

**Review by KevinPayravi — CHANGES_REQUESTED** [review]
Thanks for the submission! Need a couple small fixes, and then should be good to merge in.

File: data/sitesEN.json (hunk 1 above)
Comment (KevinPayravi) [reply data/sitesEN.json:839]: Ah, and please alphabetize the listing be moving it under the Critical Role wiki. Thank you!

Comment (VeryGreatFrog) [reply data/sitesEN.json:839]: Would it be okay to alphabeticalize it under T instead? The full name of the game is The Cycle: Frontier, not Cycle: Frontier

Comment (KevinPayravi) [reply data/sitesEN.json]: Sure, that's fine. Thank you!

**Review by KevinPayravi — APPROVED** [review]

**Pull request merged**